src/displayBackend/wayland/Compositor.cpp

```diff
@@ -46,8 +52,11 @@ Compositor::~Compositor()
 SurfacePtr Compositor::createSurface()
 {
 	LOG(mLog, DEBUG) << "Create surface";
-
+#ifdef WITH_WAYLAND_PRESENTATION_FEEDBACK
+	return SurfacePtr(new PresentationFeedbackSurface(mWlCompositor, mWlPresentation));
```

Comment:
What if setPresentation is not called (weston doesn't send presentation registry for example)?